### PR TITLE
Add new guidance page

### PR DESCRIPTION
**Pull request by amy-silcock**
Adding Results, Understanding results and conditions.

Plus some tweaks to other pages

@akmiller01 one thing to check is how the images will pull through. I've added them in a 'media' folder and linked to them in the understanding results file. This is new for us.

### CODE REVIEW

File: en/conditions.rst [hunk 1]
```diff
@@ -3,33 +3,71 @@ Conditions
...
+       If a condition relates to a whole organisation, it should not be reported here. Rather, it should be included as a document link in the organisation file.
+
+   * - `narrative <https://iatistandard.org/en/iati-standard/203/activity-standard/iati-activities/iati-activity/conditions/condition/narrative/>`__
+     - Short description of the condition.
+     -
+     - Further details about a condition can be supplied using the activity :doc:`document-link <related-documents>` element.
 
 .. meta::
-  :title: Conditions
+  :title: Results information
```
Comment (PetyaKangalova) [inline en/conditions.rst:70]: The title should still say "Conditions".

**Comment by PetyaKangalova** [comment]
Linking for reference to [IATI Discuss post](https://discuss.iatistandard.org/t/webinar-3-results-conditions-guidance/1975) where guidance pages were shared before copy-edit.

**Comment by akmiller01** [comment]
Hi @amy-silcock images aren't automatically pulled through since there are only 18 images used throughout several thousand pages, I figured it wasn't worth the time to automate their import. For any given new image, I'll just need to manually upload it to the server's media folder to be served.

If we're going to use more images in the future, it may be worth looking into streamlining that process a little.

File: en/conditions.rst (hunk 1 above)
Comment (amy-silcock) [reply en/conditions.rst]: Thanks Petya, good spot

**Comment by amy-silcock** [comment]
Thanks @akmiller01 .

We don't currently have any plans to add more images. It may depend on what type of guidance we produce and whether they'd be helpful. Maybe one to think about when the time comes?

How much work would automating this be?

**Pull request merged**